serverAkkaHttp/src/main/scala/RouterDerivation.scala

```diff
@@ -44,19 +43,17 @@ object RouterDerivationMacro extends RouterDerivationModule {
     """
   }
 
-  def deriveRouterImplPrinter[A: c.WeakTypeTag](c: Context)(a: c.Expr[A], printer: c.Expr[Printer]): c.Tree = {
```

Comment:
I can't find an official reference, but I'm pretty sure that Eugene advises to use `c.Tree` instead of `c.Expr[A]`.